src/components/data-exploration/cell-sets-tool/CellSetOperation.jsx

```diff
@@ -25,10 +25,11 @@ const CellSetOperation = (props) => {
       }}
       onCancel={() => {
         onCancel();
+        setPopoverKey(Math.random());
       }}
       key={popoverKey}
       message={helpTitle}
-      trigger='hover'
+      trigger='click'
```

Comment:
It's good to change the interaction to a click, as it is more determinant than hover. However, the user has to click and open the modal to see what each button is for.

Can you add tooltips that appears on each of the cell set operation buttons, to help the user see what each button does without having to click the buttons?

Reply:
Good catch!